dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/HttpServiceContextImpl.java

```diff
@@ -2892,8 +2905,18 @@ final protected VirtualConnection sendFullOutgoing(WsByteBuffer[] wsbb, HttpBase
             if (this instanceof HttpInboundServiceContextImpl) {
                 hisc = (HttpInboundServiceContextImpl) this;
             }
-            if (hisc != null && !(hisc.getLink() instanceof H2HttpInboundLinkWrap)) {
+            if (hisc != null && hisc instanceof HttpInboundServiceContextImpl) {
```

Comment:
Is && hisc instanceof HttpInboundServiceContextImpl  needed?  it was checked above before setting hisc